dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/client/ContributionClient.java

```diff
@@ -32,36 +28,16 @@ public interface ContributionClient extends MaatApiClient {
     @Valid
     Integer updateContributions(@RequestBody ContributionUpdateRequest contributionUpdateRequest);
 
-    @PostExchange("/prepare-fdc-contributions-files")
-    FdcGlobalUpdateResponse executeFdcGlobalUpdate();
-
-    @GetExchange("/fdc-contribution-files")
-    FdcContributionsResponse getFdcContributions(@RequestParam String status);
-
-    @PostExchange("/create-fdc-file")
-    @Valid
-    Integer updateFdcs(@RequestBody FdcUpdateRequest contributionPutRequest);
-
     @PostExchange("/log-contribution-response")
     @Valid
     Integer sendLogContributionProcessed(@RequestBody UpdateLogContributionRequest updateLogContributionRequest);
 
-    @PostExchange("/log-fdc-response")
-    @Valid
-    Integer sendLogFdcProcessed(@RequestBody UpdateLogFdcRequest updateLogFdcRequest);
-
```

Comment:
I think this one will be needed when we get to the DRC logging FDC integration tests.

Were the others all unused?

Reply:
Never mind - I see it just moved.